pages/api/id/[form]/bearer.ts

```diff
@@ -14,14 +13,14 @@ const handler = async (req: NextApiRequest, res: NextApiResponse): Promise<void>
     try {
       return await getToken(req, res);
     } catch (err) {
-      logMessage.error(err);
+      logMessage.error(err as Error);
```

Comment:
Curious is there really a need to typecast here ? Should it not already be inferred to be of type Error considering its in a catch block

Reply:
On my IDE the linter continually complains and this is because the catch block cannot be inferred as a type Error.  It is always inferred as type unknown. I don't know why the other team members linters don't complain about this?
https://stackoverflow.com/questions/69021040/why-catch-clause-variable-type-annotation-must-be-any

Reply:
Do not get the same complaints on my end. If it's in a catch block it should be inferred that it's going to be of type Error. The link you sent seems to be someone trying to typecast a specific error type

Reply:
In newer versions of typescript it's only ever inferred as `unknown`
https://kentcdodds.com/blog/get-a-catch-block-error-message-with-typescript

I can modify the `logMessage` handler instead to do strong type checking internally however this seemed more readable at the time.

Reply:
https://www.typescriptlang.org/docs/handbook/release-notes/typescript-4-0.html#unknown-on-catch-clause-bindings